tests/python_client/testcases/test_database.py

```diff
@@ -523,6 +523,95 @@ def test_using_db_not_existed(self):
         colls, _ = self.utility_wrap.list_collections()
         assert collection_w.name in colls
 
+    def test_create_same_collection_name_different_db(self):
+        """
+        target: test create same collection name in different db
+        method: 1. create a db and create 1 collection in db
+                2. create the collection in another db
+        expected: exception
+        """
+        # check default db is empty
+        self._connect()
+        assert self.utility_wrap.list_collections()[0] == []
+
+        # create a collection in default db
+        c_name = "collection_same"
+        self.init_collection_wrap(c_name)
+        assert self.utility_wrap.list_collections()[0] == [c_name]
+
+        # create a new database
+        db_name = cf.gen_unique_str("db")
+        self.database_wrap.create_database(db_name)
+        self.database_wrap.using_database(db_name)
+
+        # create a collection in new db using same name
```

Comment:
i sugesst assert there is no collection in the new db before create one.

Reply:
added. https://github.com/milvus-io/milvus/pull/28280/files#diff-e5be884a8489fffa6cff00449bff72d8a5e087ab39b64cc3acb45e096d0bfffc